src/ledger/NetworkConfig.cpp

```diff
@@ -1927,6 +1928,124 @@ SorobanNetworkConfig::evictionIterator()
 {
     return mEvictionIterator;
 }
+
+void
+SorobanNetworkConfig::updateRecalibratedCostTypesForV20(
```

Comment:
Nit: These are the cost types updated in phase 1 right? Maybe we should add a comment as to why this is here. Otherwise it's kind of confusing.

Reply:
Done.